paywall/src/unlock.js/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "unlock-checkout-builder",
```

Comment:
That's not quite the checkout only that it builds, right

Reply:
Yeah, not quite sure what the best name for this would be unless we just want to call it `unlock-paywall`

Reply:
I am actually fine with `unlock-paywall`, but what about `unlock-web` ?

Reply:
`unlock-web` sounds good to me!